infrastructure/task/scripts/redeploy-failed-deployments.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+#
+# 80% of failed deployments is fixed by redeploying the failed deployment.
+# This script will help make that easier to do.
+
+echo "Will now start to redeployg failed deployments"
+echo "Finding failed main environments"
+
+FAILED_PRODUCTION_DEPLOYMENTS=$(lagoon raw --raw "query allProjects {
+  allProjects {
+  name
+    environments(type: PRODUCTION) {
+      name
+      deployments(limit: 1) {
+        status
+        created
+      }
+    }
+  }
+}" | jq -r '.allProjects[] | .name as $name | .environments[].deployments[] | select(.status == "failed") | ($name)')
+
+echo "redeploying failed production environments"
+
+for deployment in $FAILED_PRODUCTION_DEPLOYMENTS; do
+  if [[ $deployment =~ "dpl-cms" ]]; then
+    continue
+  fi
+  echo "$deployment: deploying"
+  lagoon deploy latest -p "$deployment" -e "main" --force
+done
+
+echo "Finding failed development environments"
+
+FAILED_DEVELOPMENT_DEPLOYMENTS=$(lagoon raw --raw "query allProjects {
+  allProjects {
+  name
+    environments(type: DEVELOPMENT) {
+      name
+      deployments(limit: 1) {
+        status
+        created
+      }
+    }
+  }
+}" | jq -r '.allProjects[] | .name as $name | .environments[].deployments[] | select(.status == "failed") | ($name)')
+
+echo "redeploying failed moduletest environments"
+
+for deployment in $FAILED_DEVELOPMENT_DEPLOYMENTS; do
+  if [[ $deployment =~ "dpl-cms" ]]; then
+    continue
+  fi
+  echo "$deployment: deploying"
+  lagoon deploy latest -p "$deployment" -e "moduletest" --force
+done
```

Comment:
There is some repetition here that I wonder if we could refactor to reusable functions. I do not think it is that bad though. Keeping it simple is also nice.

Reply:
I considered that too when I wrote it. My conclusion was that this is a temporary solution to a problem we want to solve more completely, so for now let's just keep this simple